extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/ApplyRouteTargetPortConfigurator.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.kubernetes.deployment;
+
+import io.dekorate.kubernetes.decorator.NamedResourceDecorator;
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import io.fabric8.openshift.api.model.RoutePort;
+import io.fabric8.openshift.api.model.RouteSpecFluent;
+
+public class ApplyRouteTargetPortConfigurator extends NamedResourceDecorator<RouteSpecFluent<?>> {
```

Comment:
We use Configurator for other things this is a Decorator and should be named as such